pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/basicalloydbcluster/create.yaml

```diff
@@ -18,8 +18,16 @@ metadata:
   name: alloydbcluster-${uniqueId}
 spec:
   location: us-central1
-  networkRef: 
-    external: projects/${projectId}/global/networks/default
+  networkConfig:
+    networkRef: 
+      name: default
```

Comment:
Shall we use a non-default network to avoid test failures we have seen before?

Reply:
We can, but since this is a quick test (couple of minutes) it has not been seen to get into any contentions so far. the issues have been seen with long running tests